docs/src/Upsell.doc.js

```diff
@@ -46,109 +54,366 @@ card(
           '{| accessibilityLabel?: string , href?: string, label: string, onClick?: ({ event: SyntheticMouseEvent<HTMLAnchorElement> | SyntheticKeyboardEvent<HTMLAnchorElement | SyntheticMouseEvent<HTMLButtonElement> | SyntheticKeyboardEvent<HTMLButtonElement> }) => void |}, onNavigationOptions: ({ [string]: Node | ({| +event: SyntheticEvent<> |}) => void }) => void, rel: "none" | "nofollow", target: "null" | "self" | "blank" |}',
         required: false,
         defaultValue: null,
-        description: [
-          'Button to render inside the callout as the secondary call-to-action to the user. The label text has a fixed size.',
-          '- href: If href is supplied, the action will serve as a link, while if no href is supplied, the action will be a button',
-          '- label: Text to render inside the button to convey the function and purpose of the button. The button text has a fixed size.',
-          '- accessibilityLabel: Supply a short, descriptive label for screen-readers to replace button texts that do not provide sufficient context about the button component behavior. Texts like `Click Here,` `Follow,` or `Read More` can be confusing when a screen reader reads them out of context. In those cases, we must pass an alternative text to replace the button text.',
-          '- onClick: Callback fired when the button component is clicked (pressed and released) with a mouse or keyboard.',
-          `- onNavigationOptions: onNavigationOptions works in conjunction with a Provider. Pass custom props to onNavigation. See Provider for examples. onNavigation's type is flexible. Each key's value is a React.Node or an event handler function. Optional with href.`,
-          'Accessibility: `accessibilityLabel` populates aria-label. Screen readers read the `accessibilityLabel` prop, if present, instead of the button `text`.',
-        ],
-        href: '',
+        description: `
+          Secondary action for people to take on Upsell. If \`href\` is supplied, the action will serve as a link. If no \`href\` is supplied, the action will be a button.
+          The \`accessibilityLabel\` should follow the [Accessibility guidelines](#Accessibility).
+        `,
       },
       {
         name: 'title',
         type: 'string',
         required: false,
         defaultValue: null,
-        description: [
-          'Heading to render inside the Upsell above the message to convey the Upsell topic to the user. The message text has a fixed size.',
-        ],
-        href: '',
+        description: `Brief title summarizing the Upsell. Content should be [localized](/Localization).`,
       },
     ]}
   />,
 );
 
 card(
-  <Example
-    name="Simple message"
-    defaultCode={`
+  <MainSection name="Best practices">
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="lg"
+        type="do"
+        description="Use Upsells for marketing new products or encouraging upgrades."
+        defaultCode={`
+          <Upsell
```

Comment:
indentation

```
<Upsell
  title="Give $30, get $60 in ads credit"
  message="Earn $60 of ads credit, and give $30 of ads credit to a friend"
  primaryAction={{
    href: 'https://pinterest.com',
    label: 'Send invite',
    accessibilityLabel: 'Invite friend to use ads',
  }}
  dismissButton={{
    accessibilityLabel: 'Dismiss banner',
    onDismiss: () => {},
  }}
  imageData={{
    component: <Icon icon="pinterest" accessibilityLabel="" color="darkGray" size={32} />,
  }}
/>;

```

Reply:
also " vs '

Reply:
this applies to all examples. I won't copy paste reformating, mostly affects spacing/identation